.github/workflows/sync-with-PRO.jsonnet

```diff
@@ -13,10 +15,63 @@ local gha = import 'libs/gha.libsonnet';
 
 local job = {
   'runs-on': 'ubuntu-latest',
-  steps: [
+  permissions: gha.write_permissions,
+  steps: semgrep.github_bot.get_token_steps + [
      {
-       run: 'echo TODO'
+      name: 'Checkout OSS',
+      uses: 'actions/checkout@v3',
+      with: {
+        ref: '${{ github.event.repository.default_branch }}',
```

Comment:
maybe add a comment that this essentially means "develop"

Reply:
ok